content/python/concepts/built-in-functions/terms/issubclass/issubclass.md

```diff
@@ -0,0 +1,73 @@
+## **issubclass()**
+---
+Title: 'issubclass()'
+
+Description: Returns `True` <span style="color:red">if</span> a given [class](https://www.codecademy.com/resources/docs/python/classes) is a subclass of one <span style="color:red">or</span> more classes.
+
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+
+Tags:
+  - 'Classes'
+  - 'Functions'
+
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
```

Comment:
```suggestion
---
Title: 'issubclass()'
Description: 'Returns True if a given class is a subclass of one or more classes.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Classes'
  - 'Functions'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---
```

Thanks for bring in this meta-data. 😄 

There's a few more things to do here:

- We no longer need `**issubclass()**` on line # 1.
- The `Description` should be surrounded by single quotes (`' '`).
  - We can't have extra special characters or inline code here.
- No empty lines between the meta-data fields.